airflow/api_fastapi/core_api/openapi/v1-generated.yaml

```diff
@@ -2470,6 +2470,198 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPValidationError'
+  /public/dags/{dag_id}/dagRuns/{dag_run_id}/taskInstances/:
```

Comment:
Will we support wildcard ids to list all task instances for all runs or dags?

Reply:
Yes this is currently supported and tested.

Reply:
Manually added a piece of documentation for that.
![Screenshot 2024-11-07 at 10 47 27](https://github.com/user-attachments/assets/2bafdef7-6c14-4c14-af5b-7c75f2fe0852)